integration/tests/cook/test_basic.py

```diff
@@ -2043,6 +2043,39 @@ def test_user_usage_ungrouped(self):
         finally:
             util.kill_jobs(self.cook_url, job_uuids)
 
+    @unittest.skipUnless(util.are_pools_enabled(), "Requires pools")
+    @pytest.mark.serial
```

Comment:
Can we do this with a different user so we don't have to mark it serial?

Reply:
Done